GiniVision/Classes/ImageMetaInformationManager.swift

```diff
@@ -287,6 +288,13 @@ internal class ImageMetaInformationManager {
         return existingUUID ?? NSUUID().uuidString
     }
     
+    fileprivate func deviceOrientation() -> String {
+        if UIApplication.shared.statusBarOrientation.isPortrait {
```

Comment:
Another dependency on `UIApplication`...

Reply:
As I commented above, what do you suggest to solve this? I was thinking about some ideas, but none of them convinced me.